src/commands/test/lib/uitest-preparer.ts

```diff
@@ -191,13 +191,18 @@ export class UITestPreparer {
     }
 
     let testCloudPath = path.join(toolsDir, "test-cloud.exe");
+    const testCloudPathDev = path.join(toolsDir, "Xamarin.UITest.Console.exe");
 
-    if (!await fileExists(testCloudPath)) {
+    if (!await fileExists(testCloudPath) && !await fileExists(testCloudPathDev)) {
       throw new Error(`Cannot find test-cloud.exe, the exe was not found in the path specified by "--uitest-tools-dir".${os.EOL}` +
         `Please check that ${testCloudPath} points to a test-cloud.exe.${os.EOL}` +
         `Minimum required version is "${this.getMinimumVersionString()}".`);
     }
 
+    if (!await fileExists(testCloudPath)) {
```

Comment:
@owenniblock 
Double checking `!await fileExists(testCloudPath)`
I would avoid it.

Reply:
Happy for you to change it if there's a better way - I was just copying the existing pattern.